doc/index.rst

```diff
@@ -24,28 +51,43 @@ for testing.
    $DESIMODEL : data used for simulating DESI <DESIMODEL/index>
    $DESISURVEY_OUTPUT : outputs from desisurvey and surveysim <DESISURVEY_OUTPUT/index>
    $PROTODESI : data and logs from the ProtoDESI campaign <PROTODESI/index>
+
+
+Other information
+=================
+
```

Comment:
I would add one sentence here describing what types of information are described in the toctree immediately below.

Reply:
done